<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPointerDialog</class>
 <widget class="QDialog" name="EditPointerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Pointer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Address</string>
       </property>
       <property name="buddy">
        <cstring>lineEditAddress</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditAddress">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>adsfasdf</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Points to</string>
       </property>
       <property name="buddy">
        <cstring>lineEditPointsTo</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditPointsTo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>asdf</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Certainty</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxCertainty</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Author</string>
       </property>
       <property name="buddy">
        <cstring>lineEditAuthor</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditAuthor">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>asdfasdf</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Note</string>
       </property>
       <property name="buddy">
        <cstring>plainTextEditNote</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPlainTextEdit" name="plainTextEditNote">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string>asdfasdfasdf</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxCertainty"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditAddress</tabstop>
  <tabstop>lineEditPointsTo</tabstop>
  <tabstop>spinBoxCertainty</tabstop>
  <tabstop>lineEditAuthor</tabstop>
  <tabstop>plainTextEditNote</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditPointerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditPointerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
